x-pack/legacy/plugins/apm/readme.md

```diff
@@ -44,10 +41,10 @@ For testing purposes APM has invented 4 custom users:
 **kibana_write_user** Apps: read/write. Indices: None
 
 
-To create the 4 users with the correct roles run the following script:
+To create the users with the correct roles run the following script:
```

Comment:
You should make a note that Kibana needs to be running locally on localhost:5601 or `--kibanaUrl`.